client/app/components/dynamic-parameters/DynamicButton.jsx

```diff
@@ -42,7 +42,7 @@ function DynamicButton({ options, selectedDynamicValue, onSelect, enabled, stati
 
   return (
     <div ref={containerRef}>
-      <a onClick={e => e.stopPropagation()}>
+      <a role="presentation" onClick={e => e.stopPropagation()}>
```

Comment:
Why do we apply `presentation` in this specific case? It feels like this could potentially just be left as is. I'm worried that `presentation` will remove the built in semantics for the `Dropdown.button`.

Reply:
Presentation is a role reserved for elements that should be ignored by screen readers. As the role of this one is to stop bubbling, I think it's the case. Also, Eslint requires that all interactive elements have a role. Check [this](https://www.w3.org/TR/wai-aria-practices-1.1/#presentation_role).

Reply:
When we apply `presentation`, it signals the screen reader to ignore all children, too. Are we certain we want to remove the accessibility of its child components?

Reply:
Hmm... I changed my mind. [mdn](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_presentation_role#:~:text=The%20presentation%20role%20is%20used%20to%20remove%20semantic%20meaning%20from%20an%20element%20and%20any%20of%20its%20related%20child%20elements)'s documentation conflicts with the [w3 aria practices doc](https://www.w3.org/TR/wai-aria-practices-1.1/#presentation_role).

Reply:
So nevermind :) My reference was wrong.

Reply:
That caught me up as well, had to read some stuff